packages/data-store/src/index.ts

```diff
@@ -26,3 +26,11 @@ export async function createDataStore(_options: DataStoreConfigs = {}) {
 
   return true;
 }
+
+/**
+ * Close the data store connection with the database
+ *
+ */
+export async function closeDataStore() {
+  return getDataSource().destroy();
```

Comment:
do we need to explicitly issue a final commit before closing it?

Reply:
No that's not required